tests/rewards/test_reward_nets.py

```diff
@@ -541,7 +538,11 @@ def test_wrappers_pass_on_kwargs(
         env_2d.observation_space,
         env_2d.action_space,
     )
-    basic_reward_net.predict_processed = mock.Mock(return_value=np.zeros((10,)))
+    # TODO(juan) reassigning a method is very bad practice. I added
```

Comment:
You can use `with mock.patch:`

Reply:
This is gone in the latest commit to master so marking this as resolved.